frontend/src/App.test.js

```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+xtest('renders learn react link', () => {
```

Comment:
update the test ;)

Reply:
I will write an Itegration test for the signin